doc/rst/usingchapel/QUICKSTART.rst

```diff
@@ -3,18 +3,94 @@
+
+For example, on Ubuntu 24 with x86_64, you can use the following to install Chapel 2.4:
+
+.. code-block:: bash
+   wget https://github.com/chapel-lang/chapel/releases/download/2.4.0/chapel-2.4.0-1.ubuntu24.amd64.deb
+   sudo apt-get update
+   sudo agt-get install ./chapel-2.4.0-1.ubuntu24.amd64.deb
+
+For example, on Fedora 41 with aarch64, you can use the following to install Chapel 2.4:
+
+.. code-block:: bash
+   wget https://github.com/chapel-lang/chapel/releases/download/2.4.0/chapel-2.4.0-1.fc41.aarch64.rpm
+   sudo dnf update
+   sudo dnf install ./chapel-2.4.0-1.fc41.aarch64.rpm
+
+See `this page <https://chapel-lang.org/download/#linux>`_ for the most up-to-date
+list of supported OSes and platforms.
+
+brew
+~~~~
+
+If you are using `brew <https://brew.sh/>`_, you can also install Chapel directly with ``brew install chapel``. This will install the latest Chapel release available in the brew repository.
+
+Containers (Docker)
+-------------------
+
+Chapel provides a number of Docker images that can be used to run Chapel in a containerized environment. This is a good option if you want to try Chapel without installing it on your host system or if you want to ensure that you have a consistent environment across different machines.
+
+We currently provide three different Docker images: chapel, chapel-gasnet, and chapel-gasnet-smp. The `chapel` image is the most basic and includes the Chapel compiler and standard library. The `chapel-gasnet` image includes support for multi-locale (distributed memory) programming using GASNet over ethernet, while the `chapel-gasnet-smp` image includes support for multi-locale programming using shared memory.
+
+For example, the following is a minimal Dockerfile that uses the `chapel` image to run a simple Chapel program:
+
+.. code-block:: dockerfile
+
+   FROM chapel/chapel:latest
+   COPY hello.chpl /app/hello.chpl
+   WORKDIR /app
+   RUN chpl hello.chpl
+   CMD ["./hello"]
+
+Github Codespaces
+~~~~~~~~~~~~~~~~~
+
+Github Codespaces are a great way to quickly setup a develop environment for Chapel without needing to install anything on your local machine. The `chapel-hello-world <https://github.com/chapel-lang/chapel-hello-world>`_ repository provides a simple example of how to set up a Codespace for Chapel development. This repository also provides a template that you can use to create your own Codespace for Chapel development.
+
+To start using Chapel with a Codespace, use the following steps:
+
+1. Go to the `chapel-hello-world <https://github.com/chapel-lang/chapel-hello-world>`_ repository.
+2. Click on "Use this template" to create a new repository based on the template.
+3. In the new repository, click on the "Code" button, select "Codespaces", and then click on "Create codespace on main".
+
+.. image:: codespaceCreation.png
+   :width: 50%
+   :align: center
+   :alt: Screenshot of creating a Codespace for Chapel development
+
+From Source Builds via Spack
```

Comment:
I'd make this "Building From Source Via Spack" instead.  "From Source" at the start feels like it should be in quotation marks to me to separate it sufficient from "Builds", but that'd be awkward so close to the `From Source` section that shouldn't use quotation marks ...